lib/pages/member/member_management/member_management_widget.dart

```diff
@@ -515,3 +499,101 @@ Widget modalSheet(
     ],
   );
 }
+
+leaveChannel(Channel currentChannel, Account user, BuildContext context) async {
+  showDialog<String>(
+    context: context,
+    builder: (BuildContext context) => AlertDialog(
+      backgroundColor: Get.theme.colorScheme.secondaryContainer,
+      shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.all(Radius.circular(18.0))),
+      title: Text(
+        '${AppLocalizations.of(context)!.leaveChannel}  ${currentChannel.name}',
```

Comment:
`leaveChannel` should be a parameter string, such as: `"leaveChannel": "Leave channel {channelName}"`. Because for some languages, `channelName` may be in front of `Leave channel`. So, need to support internationalizing for this.

Reply:
Good point, thanks